### ISSUE TITLE
Review and standardise date and time of review pattern

### ISSUE THREAD

**Issue by kerryalyons** [DFE-Digital/dops-sas#97]
## Feedback in service
I wondered if the time should have a 'to' instead of a dash mark in it for accessibility?

## What we've seen

Across the service, sometimes we use 'to' and sometimes we use '-'. We should review this to be consistent.

## Example of using 'to'

![image](https://github.com/user-attachments/assets/b1369914-bdd8-40d8-8c61-d24b46f1fd01)

## Example of using '-' service

![image](https://github.com/user-attachments/assets/637a36b1-6354-4333-9531-7453de63c9b2)

**Comment by kerryalyons** [comment]
Look at this with Andy to see when/why it flits between the two.
